pkg/minikube/bootstrapper/bsutil/ktmpl/v1beta1.go

```diff
@@ -82,5 +82,9 @@ evictionHard:
 ---
 apiVersion: kubeproxy.config.k8s.io/v1alpha1
 kind: KubeProxyConfiguration
+clusterCIDR: "{{.PodSubnet }}"
```

Comment:
When .PodSubnet is empty, I think this line should be removed. To do so,  I think this should be:

{{ if ne .PodSubnet "" }}clusterCIDR: "{{.PodSubnet }}"{{ end }}

Reply:
I've based this config block on the existing ones, since clusterCIDR and podSubnet share  the same value

e.g: https://github.com/kubernetes/minikube/blob/master/pkg/minikube/bootstrapper/bsutil/ktmpl/v1beta1.go#L71
```go
...
networking:
  dnsDomain: {{if .DNSDomain}}{{.DNSDomain}}{{else}}cluster.local{{end}}
  podSubnet: "{{.PodSubnet }}"
  serviceSubnet: {{.ServiceCIDR}}
...
```